docs/src/integration.md

```diff
@@ -22,6 +22,11 @@ nav_order: 2
 - Swift 5.3+
 - Xcode 12.0+
 
+{: .note }
+> While the native code in the SDK is iOS 12+ compatible, the Afterpay / Clearpay checkout is a WebView wrapper for the web checkout. Browser support for this is currently iOS Safari 13.4+. It is up to the implementation to handle this difference. This allows apps that support users on iOS 12+ to use the SDK, but only users that meet the WebView requirements to access Afterpay / Clearpay.
+>
+> One method of doing so would be to hide all assets for users under iOS 13.4. This could be acheived using the `@available` attribute: `@available(iOS 13.4, *)`.
```

Comment:
```suggestion
> One method of doing so would be to hide all assets for users under iOS 13.4. This could be achieved using the `@available` attribute: `@available(iOS 13.4, *)`.
```